lms/djangoapps/discussion/rest_api/urls.py

```diff
@@ -64,7 +64,7 @@
         CourseView.as_view(),
         name="discussion_course"
     ),
-    path('v1/accounts/retire_forum', RetireUserView.as_view(), name="retire_discussion_user"),
+    path('v1/accounts/retire_forum/', RetireUserView.as_view(), name="retire_discussion_user"),
```

Comment:
It would be nice if URI can handle both with and without trailing `\`
```suggestion
    re_path(r'^v1/accounts/retire_forum/?$', RetireUserView.as_view(), name="retire_discussion_user"),
```

Reply:
done